File: docs/api/get-prices/request.cs
// Get-Prices RustOpus example
using System.Net.Http;

var authcode = "AAAAAAAA-0000-0000-0000-BBBBBBBBBBBB";
var pid = 1234;
var outFile = "get-prices.xml";

using var client = new HttpClient();
var response = await client.GetStringAsync($"https://<url-to-rustopus>/get-prices?authcode={authcode}&pid={pid}");
await File.WriteAllTextAsync(outFile, response);

File: docs/api/get-prices/request.py
'''
    Get-Prices RustOpus example
'''

from urllib import request

# - - - - - -
# PARAMETERS
# - - - - - -
url: str = "<rustopus_url_here>"
get: str = "get-prices"
authcode: str = "AAAAAAAA-0000-0000-0000-BBBBBBBBBBBB"
pid: int = 1234
outfile: str = "get-prices.xml"

request_url: str = (f"{url}/{get}"
                    f"?authcode={authcode}"
                    f"pid={pid}")

with request.urlopen(url) as response:
    with open(file = outfile,
              mode = "wb") as f:
        f.write(response.read())
